script.js

```diff
@@ -1,12 +1,125 @@
-//You can edit ALL of the code here
+const selectElement = document.getElementById("choose");
+const selectElementShow = document.getElementById("choose-show");
+const searchElement = document.getElementById("search-input");
+const numberOfEpisodes = document.getElementById("episodes-number");
+const rootElem = document.getElementById("root");
+const allEpisodes = getAllEpisodes();
+const shows = getAllShows();
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  fetch("https://api.tvmaze.com/shows/82/episodes")
```

Comment:
Excellent implementation of this. I recommend creating another js file with the getAllEpisodes() function that contains the whole JSON file for that call .Mainly so you can test it 'offline' without having to do a call to the tvmaze API.